src/ChunkMap.cpp

```diff
@@ -740,17 +740,17 @@ bool cChunkMap::DoWithChunkAt(Vector3i a_BlockPos, std::function<bool(cChunk &)>
 
 
 
-void cChunkMap::WakeUpSimulators(int a_BlockX, int a_BlockY, int a_BlockZ)
+void cChunkMap::WakeUpSimulators(const Vector3i & a_Block)
 {
 	cCSLock Lock(m_CSChunks);
 	int ChunkX, ChunkZ;
-	cChunkDef::BlockToChunk(a_BlockX, a_BlockZ, ChunkX, ChunkZ);
+	cChunkDef::BlockToChunk(a_Block.x, a_Block.z, ChunkX, ChunkZ);
```

Comment:
Perhaps not a `Vector2`, but `ChunkCoords` (doesn't it exist already?) instead, because the coords used are `x` and `z`, instead of the traditional `x` and `y`.